corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html

```diff
@@ -35,4 +35,4 @@
       </div>
   </div>
 </script>
-{% endangularjs %}
```

Comment:
i'm confused by this diff. i see the `{% endangularjs %}` in the file but it looks like it's getting (wrongly?) removed here. any idea what's happening?